docs/other-api/dev-v2.md

```diff
@@ -256,15 +256,10 @@ To use [Mock Service Worker][msw] in development, you'll need to:
 1. Run MSW as part of your app server
 2. Configure MSW to not mock internal "dev ready" messages to the dev server
 
-For example, if you are using [binode][binode] to integrate with MSW,
```

Comment:
Could you also remove the `[binode]` link definition at the bottom of this file since it would no longer be used?